scripts/irods/database_upgrade.py

```diff
@@ -69,6 +69,12 @@ def run_update(irods_config, cursor):
         database_connect.execute_sql_statement(cursor, "create index idx_data_main7 on R_DATA_MAIN (resc_id);")
         database_connect.execute_sql_statement(cursor, "create index idx_data_main8 on R_DATA_MAIN (data_is_dirty);")
 
+        timestamp = '{0:011d}'.format(int(time.time()))
+        rows = database_connect.execute_sql_statement(cursor, "select user_id from R_USER_MAIN where user_name in ('rodsadmin', 'public') and user_type = 'rodsgroup';").fetchall()
+        for row in rows:
+            group_id = row[0]
+            database_connect.execute_sql_statement(cursor, "insert into R_USER_GROUP values (?,?,?,?);", group_id, group_id, timestamp, timestamp)
+
```

Comment:
Should this code be placed here?

In other words, should it be outside of `new_schema_version` if-blocks, possibly at the bottom of the file?